index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div id="header-name">
+            <p>John Doe</p>
+        </div>
+        <ul id="items-container">
+            <li class="header-item">Portfolio</li>
+            <li class="header-item">About</li>
+            <li class="header-item">Contact</li>
+            <li class="header-item">
+                <i class="material-icons">mail_outline</i>
+            </li>
+        </ul>
+        <div id="header-menu">
+            <i class="material-icons">menu</i>
+        </div>
+    </header>
+
+    <div id="principal-container">
+        <nav id="navigation-social">
+            <div id="social-container">
+                <div class="social-logos">
+                    <img src="images/github.png" alt="github">
+                </div>
+                <div class="social-logos">
+                    <img src="images/linkedin.png" alt="linkedin">
+                </div>
+                <div class="social-logos">
+                    <img src="images/angel.png" alt="angel list">
+                </div>
+                <div class="social-logos">
+                    <img src="images/twitter.png" alt="twitter">
+                </div>
+                <div class="social-logos">
+                    <img src="images/medium.png" alt="medium">
+                </div>
+            </div>
+        </nav>
+        <section id="presentation-info">
+            <h1>Hey There. <br> I’m Alex</h1>
+            <h2>I’m a Software Developer</h2>
+            <p>
+                I can help you build a product , feature or website Look 
+                through some of my work and experience! If you like what 
+                you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+        </section>
+    </div>   
+</body>
+</html>
```

Comment:
- [ ] This is so great the efforts you made to have the same design as the one in Figma, but I can see that there are some areas where you did not follow the Figma template, Please let's use the same `fonts`, `colors`, spaces around your logo and hamburger. see the difference below 👇 

- Your portfolio
![image](https://user-images.githubusercontent.com/55465133/129260425-59554f12-eb86-49f0-b53c-309db3596023.png)

✅  The Figma template
![image](https://user-images.githubusercontent.com/55465133/129262811-c8b8ebd6-1b9a-403b-a9b3-e014de1d7075.png)

Please let's make some efforts again and make the design look like the one in the Figma template 👍